docs/source/example_notebooks/dowhy_causal_discovery_example.ipynb

```diff
@@ -98,103 +115,227 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Causal Discovery with Causal Discovery Tool (CDT)\n",
+    "# Causal Discovery with causal-learn\n",
     "\n",
-    "We use the CDT library to perform causal discovery on the Auto-MPG dataset. We use three methods for causal discovery here -LiNGAM, PC and GES. These methods are widely used and do not take much time to run. Hence, these are ideal for an introduction to the topic. Other neural network based methods are also available in CDT and the users are encouraged to try them out by themselves. \n",
+    "We use the CDT library to perform causal discovery on the Auto-MPG dataset. We use three methods for causal discovery here -PC, FCI and GES. These methods are widely used and do not take much time to run. Hence, these are ideal for an introduction to the topic. Causal-learn provides a comprehensive list of well-tested causal-discovery methods, and readers are welcome to explore.\n",
     "\n",
     "The documentation for the methods used are as follows:\n",
-    "- LiNGAM [[link]](https://fentechsolutions.github.io/CausalDiscoveryToolbox/html/_modules/cdt/causality/graph/LiNGAM.html)\n",
-    "- PC [[link]](https://fentechsolutions.github.io/CausalDiscoveryToolbox/html/_modules/cdt/causality/graph/PC.html)\n",
-    "- GES [[link]](https://fentechsolutions.github.io/CausalDiscoveryToolbox/html/_modules/cdt/causality/graph/GES.html)"
+    "- PC [[link]](https://causal-learn.readthedocs.io/en/latest/search_methods_index/Constraint-based%20causal%20discovery%20methods/PC.html)\n",
+    "- GES [[link]](https://causal-learn.readthedocs.io/en/latest/search_methods_index/Score-based%20causal%20discovery%20methods/GES.html)\n",
+    "- LiNGAM [[link]](https://causal-learn.readthedocs.io/en/latest/search_methods_index/Causal%20discovery%20methods%20based%20on%20constrained%20functional%20causal%20models/lingam.html#ica-based-lingam)"
    ]
```

Comment:
Consider adding a link to the top-level documentation to help readers explore all the other methods supported by causal-learn

Reply:
Sure, that's a great idea.